### ISSUE TITLE
Modeling change - Switch MSE to Cross Entropy (Classification Path)

### ISSUE THREAD

**Issue by vasthelium** [vasthelium/TruSic#1]
Currently we do MSE for learning loop. 

Re-implement the learning loop by replacing MSE loss with Cross-Entropy for skip-behavior classification.

Implement Cross-Entropy (Classification Path)
    * convert y → categorical (0,1,2,3)
    * add softmax output
